std/bitmanip.d

```diff
@@ -2010,6 +2013,19 @@ public:
                "00000000_00000000");
     }
 
+    @system unittest
+    {
+        // Bugzilla 17467
```

Comment:
Nit: typically this is put in front of the unittest block and there are a couple of people who prefer actual links to Bugzilla